compiler/ztests/pushdown.yaml

```
@@ -4,11 +4,11 @@ script: |
   echo === distinct
   super compile -C -O 'from file | distinct a | yield b'
   echo === fork-join
-  super compile -C -O 'fork (=> from file1 => from file2) | join on a b | yield c'
+  super compile -C -O 'fork (=> from file1 => from file2) | join on right.b=left.a | yield left.c'
```

I'd like to change this one so it references fields of both join inputs in the join condition as well as downstream in the yield.
```suggestion
  super compile -C -O 'fork (=> from file1 => from file2) | join on left.a=right.b | yield left.c,right.d'
```

---

The join condition does reference both inputs and is in purposely in reverse order to demonstrate the compiler can figure this out. I'm good with the yield though.